lua/apisix/schema_def.lua

```diff
@@ -283,6 +283,7 @@ _M.route = {
             uniqueItems = true,
         },
         desc = {type = "string", maxLength = 256},
+        priority = {type = "integer", minimum = 1},
```

Comment:
the minimum value is 1, but the default value is 0?

Reply:
Keep only the default value to 0.